lib/bas/read/postgres.rb

```diff
@@ -17,9 +17,13 @@ class Postgres < Read::Base
     def execute
       response = Utils::Postgres::Request.execute(params)
 
-      records = response.values == [] ? nil : JSON.parse(response.values.first.first)
+      unless response.values == []
+        id = response.values.first[0]
+        data = JSON.parse(response.values.first[1])
+        inserted_at = response.values.first[2]
+      end
 
-      Read::Types::Response.new(records)
+      Read::Types::Response.new(id, data, inserted_at)
```

Comment:
What will happen here if the `response.values` is `[]`? 🤔

Reply:
All of them are set as nil